calendar_reboot.js

```diff
@@ -1,4 +1,6 @@
 var masterKey = 'masterKey';
```

Comment:
It's time to pick a better name for this. How about `allCalendarIdsKey` and renaming `calendarUniqueId` to `allCalendarIds` ?